modules/extension/xsd/xsd-wmts/src/main/java/org/geotools/wmts/bindings/AcceptedFormatsTypeBinding.java

```diff
@@ -0,0 +1,72 @@
+package org.geotools.wmts.bindings;
```

Comment:
Not picking on this class in particular, but the module does not have a single test. As a supported module, it should instead. You can have a look at other OGC modules, checking the parsing tests.